Dockerfile

```diff
@@ -8,14 +8,15 @@ LABEL "com.github.actions.description"="Deploy your DNS configuration to multipl
 LABEL "com.github.actions.icon"="cloud"
 LABEL "com.github.actions.color"="yellow"
 
-ENV DNSCONTROL_VERSION="3.31.4"
-ENV DNSCONTROL_CHECKSUM="054d236531df2674c9286279596f88f02c1cf7b1448dc5f643f1a1dbe705fe8d"
+ENV DNSCONTROL_VERSION="4.11.0"
+ENV DNSCONTROL_CHECKSUM="407d5bd1b5055ad0223c7d4ff865466d423823a0abbdd8f94693b4d8c27f51e4"
```

Comment:
I'm not sure this is the right checksum. I see this in [their current checksum file](https://github.com/StackExchange/dnscontrol/releases/download/v4.11.0/checksums.txt):

        71a35bf310d0f6ec76be0e444ede6b085af0d6487b2974a657be96ac3012ec18  dnscontrol_4.11.0_linux_amd64.tar.gz